src/main/java/pl/scoutCamp/business/RankingService.java

```diff
@@ -0,0 +1,101 @@
+package pl.scoutCamp.business;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.stereotype.Service;
+import pl.scoutCamp.business.dao.TeamCategorizationSheetDAO;
+import pl.scoutCamp.business.dao.TeamDAO;
+import pl.scoutCamp.domain.Ranking;
+import pl.scoutCamp.domain.Team;
+import pl.scoutCamp.domain.TeamCategorizationSheet;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+@Slf4j
+@Service
+@AllArgsConstructor
+public class RankingService {
+
+    private final TeamDAO teamDAO;
+    private final TeamCategorizationSheetDAO teamCategorizationSheetDAO;
+    public static final String POINTS = "points";
+    public static final String NAME = "name";
+    public static final String CATEGORY = "category";
+
+    public Page<Ranking> createRankingList(
+            String period,
+            String sort,
+            String order) {
+        List<Ranking> rankingList = new ArrayList<>();
+        for (Team team : FindAllTeams()) {
+            Ranking teamSummary = Ranking.builder()
+                    .teamName(team.getName())
+                    .totalPoints(getTotalPoints(getTeamSheets(team, period)))
```

Comment:
"(getTotalPoints(getTeamSheets(team, period))" written twice, maybe let's create variable before and define it once?